pandas_tutor/marks/marks.py

```diff
@@ -339,6 +342,60 @@ def mark_for_agg(
     return row_outlines
 
 
+# df.groupby('col').apply(lambda x: x['col'].sum())
+def mark_for_groupby_apply(
+    step: GroupByApplyCall, before: EvalResult, after: EvalResult
+) -> List[Mark]:
+    #
+    if not isinstance(before, (GroupbyResult, SeriesGroupbyResult)):
+        return []
+    if not isinstance(after, (DFResult, SeriesResult)):
+        return []
+
+    groups = util.get_groups(before.val)
+    after_val = after.val
+    if isinstance(after_val.index, pd.MultiIndex):
+        # find number indices that exist in RHS
+        matches = [multi_idx[-1] for multi_idx in after_val.index]
+        # draw arrows for each pair of number index and MultiIndex from
+        # LHS to RHS
+
+        # Check to draw arrows on positional index or groups when groupby
+        # multiple columns
+        if not isinstance(after_val.index.names[-1], type(None)):
```

Comment:
this should become

```suggestion
        if after_val.index.names[-1] is not None:
```

also change elsewhere